### ISSUE TITLE
Request: Use .cue file to split current recordings (R/S type portables devices only)

### ISSUE THREAD

**Issue by Airell** [Jo2003/cd2netmd_gui#14]
I would like to de able (re)split a single existing recording with a .cue file.
Now this can only be done as a combined action with recording, but I would like to be able to only split a single recording (from .cue).

I beleive TOC Edit is only possible on R/S type portables devices, which is fine by me.

Cheers, Airell

**Comment by Jo2003** [comment]
Hi Airell,
this would only work if the cue sheet is based on a single audio file. And even then the length of the last track must be computed from the disc length and the other tracks. So, it would work, but not easy out of the box.

A better way would be a format where you give the name and the exact length of each audio track e.g. a Json file.

This can be done. If this would be OK, I can try to do this.

Best regards,
Jörg

**Comment by Airell** [comment]
That's true, for a single large file I mean. For some discs I transfer a single large file to the MD with a different program as DAO (which I really prefer because of the gapless result) doesn't always work. It looks like the disc is spinning much faster than other transfers and some MD's 'wobbles' to much with DAO it seems.
 
Couldn't the track length be calculated by subtracting the 'INDEX 01' of the current TRACK with the 'INDEX 01' of the previous TRACK from the .cue? For me, I would also be happy with an option to split at a precise mark for instance '05:42:55' (mm:ss:ms). It would also be ok to have a context sensitive mouse click on a track om the MiniDisc side to add a mark and split two tracks.

Regards, Airell

**Comment by Jo2003** [comment]
Thinking a little more about this: I don't need to know the length of the last track. The end marker is the last sector written. So, a cue file based on a single audio file should work even w/o the audio file present. I'll try to add this is a tool in settings.

**Comment by Airell** [comment]
Ok nice! If it is to 'risky' somehow, a 'split selected track at ...' would suffice to.
